modules/tensor_mechanics/test/tests/CZM/czm_3DC_load_complex.i

```diff
@@ -0,0 +1,236 @@
+[Mesh]
```

Comment:
I don't think there's any reason to have a version of this test that doesn't use the action.

Reply:
The manual setup of an nos symmetric load guarantees that the order of the dispalcemnt variable is correct in the action. if you want really want I can delete 'czm_3DC_load_noraml.i' test

Reply:
all tests are now done using `cli_args` flag